troubleshooting/ipad-multitasking-support-ios-publish-to-fail.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Invalid Bundle Error When Publishing to the App Store
+description: "Publishing to the Apple App Store fails with an ERROR ITMS-90474: Invalid Bundle. iPad Multitasking support requires these orientations."
+type: troubleshooting
+page_title: Publishing App With IPad Multitasking Support Requires All Device Orientations to Be Enabled
+slug: ipad-multitasking-support-fail-ios-publish
+position:
+tags: appbuilder, publish
+teampulseid:
+ticketid: 1029434, 1045223
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Publish - App Store</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+</table>
+
+## Description
+When you attempt to publish your app to the Apple App Store, the following error message appears: `ERROR ITMS-90474: "Invalid Bundle`.
+
+## Error Message
+`ERROR ITMS-90474: "Invalid Bundle. iPad Multitasking support requires these orientations: 'UIInterfaceOrientationPortrait, UIInterfaceOrientationPortraitUpsideDown, UIInterfaceOrientationLandscapeLeft, UIInterfaceOrientationLandscapeRight'`
+
+## Cause
+This error is observed when your app supports [iPad Multitasking](https://support.apple.com/bg-bg/HT202070) and one of the apps device orientations is disabled. By default the Multitasking feature is active and it requires all [interface orientations](https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Articles/iPhoneOSKeys.html#//apple_ref/doc/uid/TP40009252-SW10) to be enabled. 
+
+## Solution
+Based on your app requirements you can choose one of the following approaches:
+
+* If you want your app to support iPad Multitasking, you have to enable all device orientations.
+* If you do not require iPad Multitasking, you should open your apps `Info.plist` file, locate the `UIRequiresFullScreen` key, and set its value to **true**. For example: `<key>UIRequiresFullScreen</key><true/>`.
```

Comment:
Aim for uniformity in lists. This means that list items should be phrased in a similar manner. If your previous item is "If you want your app to support iPad Multitasking", this item should be "If you do not want your app to support iPad Multitasking".

Btw, I would probably go with "If you want to support iPad Multitasking in your app" as general phrasing for this. Consider if this makes sense to you as this is a suggestion and not a required change.